engines/suppliers/Gemfile.lock

```diff
@@ -2,68 +2,93 @@ PATH
   remote: .
   specs:
     suppliers (0.0.1)
-      rails (~> 4.1.7)
+      rails (~> 4.2.5)
```

Comment:
Shouldn't be here... https://github.com/coopdevs/katuma/pull/60/files is already merged, but with direct dependency ( not engine dependency), so it's a different change. Here https://github.com/coopdevs/katuma/blob/suppliers-crud/engines/suppliers/suppliers.gemspec#L19 the gemspec already says 4.2.5, but the Gemfile.lock hasn't been updated since.

My opinion: engines shouldn't have their lock file committed to the repo.